mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/114 - Staff UI

Closes #114 

To test:
- Play Physical Security scenario
- Click IT Staff button
- See one Tech staff is hired (Tom)
- Click Security to see two unhired Security staff (Dead Eye and Joey D)
- Click Dead Eye and see Hire button enabled and property panel display Dead Eye
- Click Hire and make sure matching GameObject in hierarchy is enabled and Hire/Fire buttons swap interactability
- Click Fire and make sure matching GameObject in hierarchy is disabled and Hire/Fire buttons swap interactability

Notes:
- I don't think hiring/firing events actually change game state on server correctly looking at code (hard to tell since we don't have/show the monthly budget, which would be affected by this)
